contents/docs/new-to-posthog/getting-hogpilled.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: "Getting HogPilled: how to win with PostHog"
+nextPage: ./activation.mdx
+---
+
+import { AnalyticsPlannerTip } from 'components/Docs/AnalyticsPlannerTip'
+import { Admonition } from 'components/Docs/Admonition'
+
+<AnalyticsPlannerTip />
+
+Most startups fail, and most startups that win do it by accident. 
+
+We can do better.
+
+A strong approach to product analytics lets us **win on purpose**. We define success as a number we can measure, then describe the user behavior that feeds that number.
```

Comment:
Can success be boiled down to setting a specific metric and trying to optimize for it? I feel like this isn't true as a core principle.